.ci/pitest-survival-check-xml.groovy

```diff
@@ -7,16 +7,13 @@ import groovy.util.slurpersupport.NodeChildren
 import groovy.xml.XmlUtil
 
 @Field static final String USAGE_STRING = "Usage groovy .${File.separator}.ci${File.separator}" +
-        "pitest-survival-check-xml.groovy [profile] [-g | --generate-suppression]\n" +
+        'pitest-survival-check-xml.groovy [profile]\n' +
```

Comment:
The change in quotes should be in the minor commit, but it isn't.

This PR is a bit messy and it seems very intertwined. We talked about this before with Checker and not having super PRs/Issues.

Reply:
@Vyom-Yadav, feel free to make separate PR and move it it all not functional fixed to be merged asap, and distract reviewer of this PR

Reply:
Alright, I will make a minor PR for these changes. Removed them from this PR.